src/yadism/coefficient_functions/heavy/g1_nc.py

```diff
@@ -0,0 +1,194 @@
+
+        The part proportional to ln^2(beta), :eqref:`D.15`.
+        The part proportional to ln(beta), :eqref:`D.16`.
+        The finite 'OK' (non-abelian) part, :eqref:`D.20`.
+        The finite 'QED' (abelian) part, :eqref:`D.26`.
+
+        This equation is the same as the F3 and F2 of the unpolarized case.
+        """
+
+        def cg(z, _args):
+            if self.is_below_threshold(z):
+                return 0.0
+            return (
+                self._FHprefactor
+                / z
+                * (4.0 * np.pi) ** 2
+                * (
+                    LeProHQ.cg1("x2g1", "VV", self._xi, self._eta(z))
+                    + LeProHQ.cgBar1("x2g1", "VV", self._xi, self._eta(z))
+                    * np.log(self._xi)
+                )
+            )
+
+        return RSL(cg)
+
+
+class GluonAA(GluonVV):
+    def NLO(self):
+        """
+        |ref| implements :eqref:`D.6`, :cite:`felix-thesis`.
+
+        This is the same result as GluonVV at NLO.
+
+        """
+
+        def cg(z, _args):
+            if self.is_below_threshold(z):
+                return 0.0
+            return (
+                self._FHprefactor
+                / z
+                * LeProHQ.cg0("x2g1", "AA", self._xi, self._eta(z))
+            )
+
+        return RSL(cg)
+
+    def NNLO(self):
+        """
+        |ref| implements NLO (heavy) gluon coefficient function, :cite:`felix-thesis`.
+
+        The equation is composed of the coefficient function (here denoted by cg1) :eqref:`D.14` and the scaling coefficient function (here denoted by cgBar1) :eqref:`D.38`.
+
+        The different parts of the coefficient function are as follows:
+
+        The part proportional to ln^2(beta), :eqref:`D.15`.
+        The part proportional to ln(beta), :eqref:`D.16`.
+        The finite 'OK' (non-abelian) part, :eqref:`D.23`.
+        The finite 'QED' (abelian) part, :eqref:`D.29`.
+
+        This equation is the same as the F3 of the unpolarized case.
+
+        """
+
+        def cg(z, _args):
+            if self.is_below_threshold(z):
+                return 0.0
+            return (
+                self._FHprefactor
+                / z
+                * (4.0 * np.pi) ** 2
+                * (
+                    LeProHQ.cg1("x2g1", "AA", self._xi, self._eta(z))
+                    + LeProHQ.cgBar1("x2g1", "AA", self._xi, self._eta(z))
+                    * np.log(self._xi)
+                )
+            )
+
+        return RSL(cg)
+
+
+class SingletVV(pc.NeutralCurrentBase):
+    def NNLO(self):
+        """
+        |ref| implements NLO (heavy) singlet coefficient function, :cite:`felix-thesis`.
+
+        The NLO Bethe-Heitler Quark coefficient function :eqref:`D.43` is made up of the following parts:
+
+        The part proportional to ln(beta) :eqref:`D.44`.
+        The constant part is defined in :eqref:`D.48`.
+
+        The NLO Bethe-Heitler quark scaling coefficient functions :eqref:`D.51` is denoted by cqBarF1.
```

Comment:
```suggestion
        The NLO Bethe-Heitler quark scaling coefficient functions :eqref:`D.51` is denoted by :mod:`cqBarF1`.
```
It would be better to highlight for clarity.

Reply:
let's keep `mod` for what it is intended - modules (else you'll get a Sphinx error) - you can use two backticks (sorry I can't type - or I don't know how to trick github :upside_down_face: )

Reply:
Overlooked! You could actually backslash :-)

Reply:
\`\`

```
``var``
```

:P